src/components/ChecFilterBar.vue

```diff
@@ -164,12 +179,24 @@ export default {
        * Indicates that the user has requested the filters be changed
        */
       this.$emit('change-filters', filters);
+      if (filters.length === 0 && this.activeSearch.length > 0) {
+        this.clearSearch();
+      }
+    },
+    clearSearch() {
+      /**
+       * Indicates that the user has requested the search (that has been performed) should be cleared
+       */
+      this.$emit('clear-search');
     },
     doSearch() {
       /**
        * Indicates that the user has requested a search should be performed with the current search term
        */
       this.$emit('search');
+
+      // Blur the search field to hide the popover
+      this.$refs.search.$el.querySelector('input').blur();
```

Comment:
Are we sure that this ref will exist? If not, do you think it's worth adding an if for type safety around it?

Reply:
Yeah it'll always exist.